lib/icinga/clusterevents.cpp

```diff
@@ -24,6 +24,7 @@ INITIALIZE_ONCE(&ClusterEvents::StaticInitialize);
 
 REGISTER_APIFUNCTION(CheckResult, event, &ClusterEvents::CheckResultAPIHandler);
 REGISTER_APIFUNCTION(SetNextCheck, event, &ClusterEvents::NextCheckChangedAPIHandler);
+REGISTER_APIFUNCTION(SetLastCheckStarted, event, &ClusterEvents::LastCheckStartedChangedAPIHandler);
```

Comment:
Please add documentation about this in [19-technical-concepts.md#json-rpc-message-api](https://icinga.com/docs/icinga2/latest/doc/19-technical-concepts/#json-rpc-message-api)